File: Assets/ImageEffects/Classes_C#/DepthOfField.cs
using UnityEngine;
//using IndieEffects;
//using IndieEffects_Depth;

[RequireComponent(typeof(IndieEffects))]
[RequireComponent(typeof(IndieEffects_Depth))]
public class DepthOfField : MonoBehaviour
{
    public Shader shader;
    private Material DOFMat;
    [Range (0f,20f)]
    public float FStop;
    [Range (0f,2.5f)]
    public float BlurAmount;

    private void Start () 
    {
	    DOFMat = new Material(shader);
    }

    private void Update () 
    {
        DOFMat.SetTexture("_MainTex", IndieEffects.renderTexture);
        DOFMat.SetTexture("_Depth", IndieEffects_Depth.DepthTex);
	    DOFMat.SetFloat ("_FStop", FStop/10);
	    DOFMat.SetFloat ("_Amount", BlurAmount);
    }

    private void OnPostRender()
    {
        IndieEffects.FullScreenQuad(DOFMat);
    }
}

File: Assets/ImageEffects/Classes/DepthOfField.js
#pragma strict

@script RequireComponent(IndieEffects)
@script RequireComponent(IndieEffects_Depth)
import IndieEffects;
import IndieEffects_Depth;

var shader : Shader;
private var DOFMat : Material;
@range (0,20)
var FStop : float;
@range (0,2.5)
var BlurAmount : float;

function Start () {
	DOFMat = new Material(shader);
}

function Update () {
	DOFMat.SetTexture("_MainTex",renderTexture);
	DOFMat.SetTexture("_Depth",DepthTex);
	DOFMat.SetFloat ("_FStop", FStop/10);
	DOFMat.SetFloat ("_Amount", BlurAmount);
}

function OnPostRender () {
	FullScreenQuad(DOFMat);
}